ovos_utils/sound/__init__.py

```diff
@@ -32,110 +33,86 @@ def _get_pulse_environment(config):
     else:
         return os.environ
 
-
-def play_acknowledge_sound():
-    """Acknowledge a successful request.
-
-    This method plays a sound to acknowledge a request that does not
-    require a verbal response. This is intended to provide simple feedback
-    to the user that their request was handled successfully.
-    """
+def _play_specific_sound(sound_name, bus=None):
     audio_file = resolve_resource_file(
-        read_mycroft_config().get('sounds', {}).get('acknowledge'))
+        read_mycroft_config().get('sounds', {}).get(sound_name))
 
     if not audio_file:
-        LOG.warning("Could not find 'acknowledge' audio file!")
+        LOG.warning("Could not find '%s' audio file!", sound_name)
         return
 
-    process = play_audio(audio_file)
+    process = play_audio(audio_file, bus=bus)
```

Comment:
takes a URI, but is being given a string representing a path

Reply:
agreed - but I just added the bus arg :)

Reply:
well, this takes whatever resolve_resource_file supports

historically this was either a full path or a partial path to be checked in standard dirs  (res folder in core, but also in XDG level now)

out of scope for this PR to change this, resolve_resource_file converts this into a full path or None

Reply:
Resolved assuming the answer in the other convo is "yes it's working"